### ISSUE TITLE
Summarizing possible issues

### ISSUE THREAD

**Issue by KatelynChiang** [ErikaCanda/Final_Project4#9]
Your github is missing "output" and "report" folders... I wonder if this is the root cause of all the issues. I also wonder if "pacman::p_load" may be helpful for when you use packages.

**Comment by KatelynChiang** [comment]
FYI - Here are all the messages I got when running the make commands now, but it worked after installing janitor and config :) 

Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ ls
Makefile                data
README.md               output
code                    project4_report.Rmd
config.yml              report
Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ make output/data_clean.rds
Rscript code/00_clean_data.R
here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika
Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ make output/table_one.rds
Rscript code/01_make_table1.R
here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika
3 observations missing `FIPAG water connection` have been removed. To include these observations, use `forcats::fct_na_value_to_level()` on `FIPAG water connection` column before passing to `tbl_summary()`.
Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ make output/barplot.png
Rscript code/02_make_barplot.R
Error in library(janitor) : there is no package called ‘janitor’
Execution halted
make: *** [output/barplot.png] Error 1
Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ make output/barplot.png
Rscript code/02_make_barplot.R

Attaching package: ‘janitor’

The following objects are masked from ‘package:stats’:

    chisq.test, fisher.test

here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika
Scale for fill is already present.
Adding another scale for fill, which will replace the existing scale.
Saving 7 x 7 in image
Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ make project4_report.config_default.html
Rscript code/02_make_barplot.R

Attaching package: ‘janitor’

The following objects are masked from ‘package:stats’:

    chisq.test, fisher.test

here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika
Scale for fill is already present.
Adding another scale for fill, which will replace the existing scale.
Saving 7 x 7 in image
Rscript code/03_render_report.R
here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika
Error in loadNamespace(x) : there is no package called ‘config’
Calls: loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
make: *** [project4_report.config_default.html] Error 1
Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $ make project4_report.config_default.html
Rscript code/02_make_barplot.R

Attaching package: ‘janitor’

The following objects are masked from ‘package:stats’:

    chisq.test, fisher.test

here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika
Scale for fill is already present.
Adding another scale for fill, which will replace the existing scale.
Saving 7 x 7 in image
Rscript code/03_render_report.R
here() starts at /Users/Katelyn/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika


processing file: project4_report.Rmd
                                                                                                             
output file: project4_report.knit.md

/Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/pandoc +RTS -K512m -RTS project4_report.knit.md --to html4 --from markdown+autolink_bare_uris+tex_math_single_backslash --output report/project4_report_config_.html --lua-filter /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/rmarkdown/rmarkdown/lua/pagebreak.lua --lua-filter /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/rmarkdown/rmarkdown/lua/latex-div.lua --embed-resources --standalone --variable bs3=TRUE --section-divs --template /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable theme=bootstrap --mathjax --variable 'mathjax-url=https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' --include-in-header /var/folders/3v/kx_7j1hd45v6p0sp7s5x7_0w0000gn/T//RtmpYBZOdR/rmarkdown-str103c73b951bf5.html 

Katelyn's 🪩 ~/Library/CloudStorage/OneDrive-EmoryUniversity/DATA-550/Final_Project/Peer_Review/Erika_Try2/PeerReview_Erika $
